plugins/casutg/src/test/java/kg/apc/jmeter/threads/AbstractSimpleThreadGroupImpl.java

```diff
@@ -13,4 +14,9 @@ public void scheduleThread(JMeterThread thread, long now) {
     public Map<JMeterThread, Thread> getAllThreads() {
         return this.allThreads;
     }
+
+    @Override
```

Comment:
Same here - can AbstractSimpleThreadGroup bother implementing this, so the implementation classes don't have to?